src/main/java/Hello.java

```diff
@@ -1,5 +1,6 @@
 public class Hello {
     public static void main(String[] args) {
         System.out.println("Hello World!");
+        System.out.println("change");
```

Comment:
```suggestion
        System.out.println("change");
```System.out.print("change");